Dockerfile

```diff
@@ -9,10 +9,12 @@ RUN dnf -y update
 # enum34: rpmdb State
 # whichcraft: shutil.which() for py2
 # jinja2: rpmspec template
-RUN dnf -y install httpd python2-behave python2-six python-enum34 https://kojipkgs.fedoraproject.org//packages/python-whichcraft/0.4.0/2.fc24/noarch/python2-whichcraft-0.4.0-2.fc24.noarch.rpm python-jinja2
+RUN dnf -y install dnf-plugins-core python3-dnf-plugins-core python2-dnf-plugins-core httpd python2-behave python2-six python-enum34 https://kojipkgs.fedoraproject.org//packages/python-whichcraft/0.4.0/2.fc24/noarch/python2-whichcraft-0.4.0-2.fc24.noarch.rpm python-jinja2
 COPY dnf-docker-test/repo /var/www/html/repo/
 COPY dnf-docker-test/features /behave/
 
+# Allows to run test with rpms from only single component in rpms/
+RUN dnf -y copr enable rpmsoftwaremanagement/dnf-nightly
```

Comment:
When we will install packages for `/rpms/` then it will mess up which is not what we want.

About single packages... You can download latest RPMs from COPR, put them into `rpms/` and replace packages which you want. This way we ensure that nothing will break your test environment.